setup.py

```diff
@@ -2,12 +2,15 @@
 
 from distutils.extension import Extension
 from Cython.Distutils import build_ext
+# to get the proper numpy header locations (if using anaconda)
+import numpy as np
 
 ext_modules = []
 ext_modules.append(Extension(
```

Comment:
What would you think about moving the BP stuff out into a separate repository? To me, it just doesn't really seem to fit here because it's not specific to the KITTI dataset at all.